<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>adminwindow</class>
 <widget class="QDialog" name="adminwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1205</width>
    <height>710</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>1181</width>
     <height>671</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>1181</width>
     <height>671</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="stadiumPg">
    <property name="minimumSize">
     <size>
      <width>1179</width>
      <height>669</height>
     </size>
    </property>
    <widget class="QFrame" name="stadiumSelectFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>361</width>
       <height>521</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QLabel" name="stadiumSelectLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select Stadium:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QListWidget" name="stadiumSelectWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addStadBtn">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Add a Stadium</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="leagueSelectFrame">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>361</width>
       <height>71</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QComboBox" name="leagueSelectComboBox">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="leagueSelectLabel">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Select League:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="frame">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>20</y>
       <width>741</width>
       <height>621</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <widget class="QPushButton" name="saveBtn">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>560</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="text">
       <string>Save Changes</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="backBtn">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>560</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>701</width>
        <height>521</height>
       </rect>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="stadEditTab">
       <attribute name="title">
        <string>Stadium Edit</string>
       </attribute>
       <widget class="QLineEdit" name="stadNameEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>90</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="stadNameLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Stadium Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="teamNameLabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>70</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Team Name:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="teamNameEdit">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>90</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="streetAddressLavel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Street Address:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="streetAddressEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>160</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="cityNameEdit">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>160</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="cityNameLabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>140</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>City Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="stateLabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>190</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>State:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="zipCodeEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>260</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="zipCodeLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>240</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Zip Code:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="phoneNumEdit">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>260</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="phoneNumLabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>240</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Phone Number:</string>
        </property>
       </widget>
       <widget class="Line" name="lineStadToAddress">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>110</y>
          <width>631</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="Line" name="lineAddressToExtra">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>290</y>
          <width>631</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="openDateLabel">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>310</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Opening Date:</string>
        </property>
       </widget>
       <widget class="Line" name="lineExtraToCapacity">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>360</y>
          <width>631</width>
          <height>16</height>
         </rect>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QComboBox" name="stateComboBox">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>210</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="countryLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>190</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Country:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="countryComboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>210</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="maxCapacity">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>400</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="maxCapacityFrameEdit">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>400</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="capacityLabel_1">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>380</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Max Capacity:</string>
        </property>
       </widget>
       <widget class="QLabel" name="capacitySeason_1">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>380</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Season/Time Frame:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="minCapacity">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>450</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="capacitySeason_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>430</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Season/Time Frame:</string>
        </property>
       </widget>
       <widget class="QLabel" name="capacityLabel_2">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>430</y>
          <width>141</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Min Capacity:</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="minCapacityFrameEdit">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>450</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="leagueEditLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>League:</string>
        </property>
       </widget>
       <widget class="QComboBox" name="leagueEditComboBox">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="grassLabel">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>310</y>
          <width>131</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>Grass Type</string>
        </property>
       </widget>
       <widget class="QComboBox" name="grassEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>330</y>
          <width>291</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="dateEdit">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>330</y>
          <width>291</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="souvEditTab">
       <attribute name="title">
        <string>Souvenir Edit</string>
       </attribute>
       <widget class="QTableView" name="souvenirTableEdit">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>311</width>
          <height>391</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="souvenirEditLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Souvenir Edit Table:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="souvNameLineEdit">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>130</y>
          <width>271</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="souvPriceLineEdit">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>210</y>
          <width>271</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="nameLabel">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>100</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Item Name:</string>
        </property>
       </widget>
       <widget class="QLabel" name="priceLabel">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>180</y>
          <width>71</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Price:</string>
        </property>
       </widget>
       <widget class="QLabel" name="instruction1">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>151</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>1. Select Item To Change</string>
        </property>
       </widget>
       <widget class="QLabel" name="instruction2">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>70</y>
          <width>211</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>2. Edit Values</string>
        </property>
       </widget>
       <widget class="QPushButton" name="updateBtn">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>350</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Update with Edits</string>
        </property>
       </widget>
       <widget class="QPushButton" name="addNewBtn">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>270</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Add New</string>
        </property>
       </widget>
       <widget class="QPushButton" name="deleteSouv">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>430</y>
          <width>271</width>
          <height>51</height>
         </rect>
        </property>
        <property name="text">
         <string>Delete Selected Item</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="EdgesEdit">
       <attribute name="title">
        <string>Edges Edit</string>
       </attribute>
       <widget class="QTableView" name="adjMatrix">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>671</width>
          <height>451</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>15</x>
          <y>10</y>
          <width>161</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Adjacency Matrix</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="addNewStadBtn">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>560</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="toolTipDuration">
       <number>-1</number>
      </property>
      <property name="text">
       <string>Add New Stadium</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <zorder>frame</zorder>
    <zorder>stadiumSelectFrame</zorder>
    <zorder>leagueSelectFrame</zorder>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
